argilla/tests/unit/test_io/test_hf_datasets.py

```diff
@@ -46,6 +46,7 @@ def test_to_datasets_with_partial_values_in_records(self):
 
         ds = HFDatasetsIO.to_datasets(records)
         assert ds.features == {
+            "status": Value(dtype="string", id=None),
```

Comment:
perhaps the naming can be more explicit? as a random person's "status" might not be explicit enough. Also, what is the status without n-required responses? will it always be pending because this might lead people to filter on that assuming there are no responses?

Reply:
`status` is a calculated value based on the responses so not sure if this value should be persisted in that way.

Reply:
I mean that the status for the record being done or not, is a bit unclear, and might have a more explicit name, like “required_n_annotation_status”. Also because we’ve got a so-called response-level status. 

Additionally, I noticed we will enforce a default 1 min_annotation per record but does this imply we will have a max_annotation at some point? I think n_required_responses might be a better name?

I thought it would be weird to have all records in pending status is we would not enforce a default, but given that it is set to 1, I guess it does not matter. 

Lastly, I think the status I recomputed when doing from_disk  or from_hub and you upload the records, right? So I would say the status for records is more of a state that lives within the specific Argilla instance and dataset that on the hub or disk.

Reply:
I would not persist it or rename it when it's persisted, because it would have confusing effects that are difficult to get round and a bit opaque. As you say, It's result of a function of distributions settings & responses, and not attribute.